.chloggen/elasticsearchexporter_otel-mode-metrics.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add OTel mapping mode for metrics
```

that's a bit mysterious, can you add to what mapping we map to?
```suggestion
note: Add support for Elastic ECS data model mapping from OpenTelemetry metrics. See <URL> for the mapping.
```

---

It is not related to ECS data model. See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34472/files#r1718710444 for explanation.